datajunction-server/datajunction_server/construction/build.py

```diff
@@ -281,6 +281,22 @@ def _build_tables_on_select(
         )  # got a materialization
         if node_table is None:  # no materialization - recurse to node first
             node_query = parse(cast(str, node.query))
+
+            dj_current_timestamp = list(
```

Comment:
In Slack, I had asked if this should be more generic where there is a function registry dedicated to dj runtime functions. Then this would just happen upon rendering the ast via `Function.__str__`

Reply:
Good point! This was your comment on Slack right?

> So, built-in macros or functions that run at the time of rendering a query? Look up a function, see it is a runtime one, when the query is rendered the builtin gives a string

I like this... let me see if I can make this setup more generic.